### PR TITLE
doc: Bluetooth samples: fix tool references

### PR DESCRIPTION
**Pull request by peknis**
NCSDK-30523
Changing testing tool links from nRF Connect for Desktop to Bluetooth Low Energy app.

### CODE REVIEW

File: samples/bluetooth/central_and_peripheral_hr/README.rst [hunk 1]
```diff
@@ -36,7 +36,7 @@ You can also mix devices when testing this sample.
 For a simple echo test, you only need one additional device.
 Alternatively, you can use a smartphone providing the HRS functionality and a development kit running the :zephyr:code-sample:`ble_central_hr` sample.
 
-For testing, you can also use `nRF Connect for Desktop`_.
+For testing, you can also use `Bluetooth Low Energy app`_.
```
Comment (wiba-nordic) [inline samples/bluetooth/central_and_peripheral_hr/README.rst:39]:
```suggestion
For testing, you can also use the `Bluetooth Low Energy app`_.
```

**Review by wiba-nordic — APPROVED** [review]
Approving with a minor nit.

**Review by annwoj — APPROVED** [review]

File: samples/bluetooth/central_hids/README.rst [hunk 2]
```diff
@@ -207,15 +207,15 @@ Testing with nRF Connect for Desktop
       1. |connect_terminal_specific|
```
Comment (kapi-no) [inline samples/bluetooth/central_hids/README.rst:207]: Shouldn't we change the name of the test section?

```
Testing with nRF Connect for Desktop
```

->

```
Testing with Bluetooth Low Energy app
```

to reflect the change?

That's how I did it in my HIDS mouse sample PR. Should we align the approach?

https://github.com/nrfconnect/sdk-nrf/pull/19039/

Comment (kapi-no) [reply samples/bluetooth/central_hids/README.rst:207]: I see that you also used the new section name in the HIDS keyboard PR

**Review by kapi-no — COMMENTED** [review]
Reviewed the HIDS sample part

File: samples/bluetooth/central_hids/README.rst (hunk 2 above)
Comment (kapi-no) [reply samples/bluetooth/central_hids/README.rst:207]: This comment is also relevant for the following samples:

- samples/bluetooth/central_and_peripheral_hr/README.rst
- samples/bluetooth/peripheral_power_profiling/README.rst

Comment (peknis) [reply samples/bluetooth/central_hids/README.rst:207]: Good catch. Fixed now.

Comment (kapi-no) [reply samples/bluetooth/central_hids/README.rst:207]: thanks, resolving

**Review by kapi-no — APPROVED** [review]

File: samples/bluetooth/central_and_peripheral_hr/README.rst [hunk 3]
```diff
@@ -36,7 +36,7 @@ You can also mix devices when testing this sample.
 For a simple echo test, you only need one additional device.
 Alternatively, you can use a smartphone providing the HRS functionality and a development kit running the :zephyr:code-sample:`ble_central_hr` sample.
```
[the remark this answers is not in the document]
Comment (peknis) [reply samples/bluetooth/central_and_peripheral_hr/README.rst:37]: Why not.

File: samples/bluetooth/peripheral_hids_keyboard/README.rst [hunk 4]
```diff
@@ -203,10 +203,10 @@ To test with a Microsoft Windows computer that has a Bluetooth radio, complete t
       #. Disconnect the computer from the device by removing the device from the computer's devices list.
 
 
-Testing with nRF Connect for Desktop
-------------------------------------
+Testing with Bluetooth Low Energy app
```
[the remark this answers is not in the document]
Comment (peknis) [reply samples/bluetooth/peripheral_hids_keyboard/README.rst:206]: Kamil did that in his PR related to HIDS mouse sample.

[the remark this answers is not in the document]
Comment (divipillai) [reply samples/bluetooth/peripheral_hids_keyboard/README.rst:206]: oh yeah :)

File: samples/bluetooth/central_and_peripheral_hr/README.rst [hunk 5]
```diff
@@ -36,7 +36,7 @@ You can also mix devices when testing this sample.
...
 
-For testing, you can also use `nRF Connect for Desktop`_.
+For testing, you can also use the `Bluetooth Low Energy app`_.
```
[the remark this answers is not in the document]
Comment (peknis) [reply samples/bluetooth/central_and_peripheral_hr/README.rst:39]: Done.

**Review by divipillai — APPROVED** [review]

**Pull request merged**